docs/documentation/creating-routes.md

```diff
@@ -25,11 +25,11 @@ Let's break this down into bits:
 * **template** : the 1st parameter sent to response.render, the name of the template file used to render the page, minus its `.html` extension
 * **data** : [optional] the 2nd parameter sent to response.render, an object containing variables to send into the template
 
-So as an example, a request for the URL `http://localhost:3000/examples/template-data` has this route:
+So as an example, a request for the URL `http://localhost:3000/docs/examples/template-data` has this route:
```

Comment:
this is only works in the separate `documentation_routes.js` file, which hasn't been introduced as a concept. Earlier this doc talks about `routes.js` instead. I'd avoid this issue altogether by not referring to `docs` or `examples` in this example